build.gradle

```diff
@@ -5,7 +5,7 @@ plugins {
 }
 
 group = 'cuchaz'
-version = '0.15.3'
+version = '0.15.4'
```

Comment:
0.16.0? This adds new features.

Reply:
yes, needs to be 0.16.0 please.

Reply:
What was I thinking? You even said "minor version bump" lol